### ISSUE TITLE
Display mode toggle

### ISSUE THREAD

**Issue by zadam** [isaul32/ckeditor5-math#16]
Hi, thanks for the autoformatting and recent fixes! I really appreciate them.

I have a small issue/question...

For me the name "display mode" is quite confusing - although I know now what does it do, I still don't know why it is named "display mode". In CKEditor terminology this would be called "block mode" (as opposed to "inline mode") which IMHO reflects the usage quite well.

Also I would propose a radio button toggle which would make it clear what are the 2 options:

![](https://user-images.githubusercontent.com/617641/98451025-4ee32800-2142-11eb-9efc-3f158503c528.png)

**Comment by isaul32** [comment]
I agree, that toggle button can be a little bit confusing.

I have used [official UI library components](https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-ui) for a popup form and I think there is no better component available. So implementing a radio button requires some extra work.

I chose the term display because it's used in KaTex (displayMode), MathJax (displayed) and LaTeX (displaymath) documentations.
